src/context/FilterProjectsContext.jsx

```diff
@@ -577,63 +576,53 @@ export const FilterProjectsProvider = ({ children }) => {
   return (
     <FilterProjectsContext.Provider
       value={{
-        URL_PARAMS,
```

Comment:
I also alphabetized so that I could more easily triple check before removing items